### DISCUSSION TITLE
error question

### DISCUSSION THREAD

**Discussion by sidesharkdev** [community/community#142862] [Other Feature Feedback, Questions, & Ideas] [answered]

**Comment by Salviomcka** [comment, marked as answer]
Ok

Em 28/10/2024 22:32, "TEAM ERROR" ***@***.***> escreveu:

Closed #142862 <https://github.com/orgs/community/discussions/142862> as
resolved.

—
Reply to this email directly, view it on GitHub
<https://github.com/orgs/community/discussions/142862>, or unsubscribe
<https://github.com/notifications/unsubscribe-auth/A3CCMUAZBNSEP27H4OQXD43Z52UP3AVCNFSM6AAAAABQXWII4SVHI2DSMVQWIX3LMV45UABFIRUXGY3VONZWS33OIV3GK3TUHI5E433UNFTGSY3BORUW63R3GE2TONZYGEYA>
.
You are receiving this because you are subscribed to this thread.Message
ID: <community/community/repo-discussions/142862/discussion_event/1577810@
github.com>

**Reply by sidesharkdev** [reply]
I would like to delete my post in the GitHub Discussions because I made a mistake in my writing. Can you please assist me with this?

**Comment by NoNameErr** [comment]
You’re an error, and the question is an error too! 😂
